https://github.com/influxdata/telegraf/pull/8350

commit 97c6c8f0c1e7e26113944c7dd8807c839aebada1
Author: Konstantin Kulikov <k.kulikov2@gmail.com>
Date:   Mon Nov 2 15:24:09 2020 +0300

    fix plugins/input/ras test
    
    On some systems /tmp/test.db happens to exist which
    makes this test fail.

diff --git a/plugins/inputs/ras/ras_test.go b/plugins/inputs/ras/ras_test.go
index b8b70d55..a90258bb 100644
--- a/plugins/inputs/ras/ras_test.go
+++ b/plugins/inputs/ras/ras_test.go
@@ -114,7 +114,7 @@ func TestMultipleSockets(t *testing.T) {
 func TestMissingDatabase(t *testing.T) {
 	var acc testutil.Accumulator
 	ras := newRas()
-	ras.DBPath = "/tmp/test.db"
+	ras.DBPath = "/nonexistent/ras.db"
 	err := ras.Start(&acc)
 	assert.Error(t, err)
 }
